app/(main)/markets/News.tsx

```diff
@@ -47,11 +50,11 @@ export const News = ({ id }: NewsProps) => {
   if (!news) return <div className="py-4 text-center">No news</div>;
 
   return news.map((newsArticle, index) => (
-    <NewsArticle key={index} url={newsArticle.url} title={newsArticle.title} />
+    <NewsArticle id={id} key={index} url={newsArticle.url} title={newsArticle.title} />
   ));
 };
 
-const NewsArticle = ({ url, title }: NewsArticleProps) => {
+const NewsArticle = ({ id, url, title }: NewsArticleProps) => {
```

Comment:
Don't think we're using id anymore
```suggestion
const NewsArticle = ({ url, title }: NewsArticleProps) => {
```